### PR TITLE
Adding Pull Request Template.

### PR DESCRIPTION
**Pull request by phil-plencner-hl**
Does this need to be put into the main branch for it to actually get picked up by Github?

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# What does this Pull Request do?
+A brief description of what the intended result of the PR will be and/or what problem it solves.
+
+# How should this be tested?
+
+A description of what steps someone could take to:
+* Reproduce the problem you are fixing (if applicable)
+* Test that the Pull Request does what is intended.
+* Please be as detailed as possible.
+* Good testing instructions help get your PR completed faster.
```
Comment (awoods) [inline .github/pull_request_template.md:17]: Per the feedback from @ives1227 , could we add a section here along the lines of:
```
# Test coverage
Yes/No: Are changes in this pull-request covered by:
   - unit tests?
   - integration tests?
```

**Review by awoods — CHANGES_REQUESTED** [review]

**Comment by awoods** [comment]
Can we target this PR to `main`?

**Comment by phil-plencner-hl** [comment]
@awoods: I committed the Test Coverage changes to the file and they are part of this PR.

I discussed with @enriquediaz earlier today, and we're planning to merge everything from development branch into main after the demo on Monday. Perhaps this can be part of that PR to main....

File: .github/pull_request_template.md (hunk 1 above)
Comment (phil-plencner-hl) [reply .github/pull_request_template.md:17]: Done!

**Review by awoods — APPROVED** [review]
Looks great. Thanks, @phil-plencner-hl .

Merging into `main` after the demo makes sense.

**Pull request merged**